.travis.yml

```diff
@@ -116,21 +120,33 @@ matrix:
         # location during testing. We also use this build to make sure that the
         # dependencies get correctly installed with pip.
         - os: linux
+          arch: amd64
           env: SETUP_CMD='test --remote-data=astropy --readonly'
                LC_CTYPE=C.ascii LC_ALL=C
                PIP_DEPENDENCIES="" CONDA_DEPENDENCIES=""
                INSTALL_WITH_PIP=True
                EXTRAS_INSTALL="test,all"
 
         - os: linux
+          arch: amd64
           stage: Initial tests
           env: PYTHON_VERSION=3.7 CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
                PIP_DEPENDENCIES="scikit-image $ASDF_PIP_DEP"
                SETUP_CMD='test -a "--durations=50"'
           compiler: clang
 
+        # arm64
+        - os: linux
+          arch: arm64
+          stage: Arm tests
+          env: SETUP_CMD='test'
+          install:
+             - sudo apt-get -y install python3-venv python3-pip python3-numpy python3-matplotlib python3-kiwisolver python3-coverage python3-sgp4 python3-psutil python3-objgraph python3-ipython python3-graphviz 
```

Comment:
I meant pip installing the bare minimum python dependencies them not apt-get install. That should fit in the timeout range, and hopefully still lets enough time for building astropy and testing it.

Reply:
I thought we could get away with using a system numpy but it wants to compile its own. A version thing or is it in some sort of virtualenv?

Reply:
I'm not sure, we never suggest to use system packages but stick with pip (or conda if one must).

Reply:
oh certainly under any normal circumstances I've given up on anything outside a virtualenv. But we're in some kind of container here, and it seemed worth a try.